README.md

```diff
@@ -257,6 +259,16 @@ Notes:
 
 ### Special classes within teacher content
 
+####Teacher Content with NGSS tag
+
+```html
+
+<note class="os-teacher">
+  <label>Teacher Edition</label>
+  <p>NGSS HS-PS2-1:<span class="ost-tag-ngss-k12phys-hs-ps2-1 ost-ngss-def">Students who demonstrate understanding can: Analyze
```

Comment:
ok, It just means the UX will need to decide what the human-readable version of the tag is. Either `NGSS HS-PS2-1` or `Students who demonstrate understanding can: Analyze ...`